src/IronWorker.php

```diff
@@ -76,10 +76,11 @@ public function __construct($config = null)
      * @return bool Result of operation
      * @throws \Exception
      */
-    public function upload($directory, $run_filename, $code_name, $options = array())
+    public function upload($directory, $run_filename, $code_name, $options = array(), $ignored = [])
```

Comment:
well.... first of all, you're right about options being sent to a server. adding `$ignored` in there might be not the best idea.
But that's not necessary true - you could simply extract & delete  $options['ignored'] and use it locally without sending it to a server, i see similar pattern in multitude of ruby libraries.